flytekit/core/container_task.py

```diff
@@ -89,16 +90,8 @@ def __init__(
     def resources(self) -> ResourceSpec:
         return self._resources
 
-    def execute(self, **kwargs) -> Any:
-        print(kwargs)
-        env = ""
-        for k, v in self.environment.items():
-            env += f" -e {k}={v}"
-        print(
-            f"\ndocker run --rm -v /tmp/inputs:{self._input_data_dir} -v /tmp/outputs:{self._output_data_dir} {env}"
```

Comment:
That'd be an interesting feature, we have to assume that the docker client is configured properly (which I think it's a fair assumption). In order for this to happen we have to solve a few problems:
1. literals and promise generation. This can be easily done in `local_execute`.
2. Detect some features that we can't implement locally, for example pod templates.

Reply:
I opened https://github.com/flyteorg/flyte/issues/3876 to track this work.